tests/x11/yast2_lan_restart.pm

```diff
@@ -90,9 +90,13 @@ sub run {
     unless (is_network_manager_default) {
         # Run detailed check only if explicitly configured in the test suite
         check_etc_hosts_update() if get_var('VALIDATE_ETC_HOSTS');
-        verify_network_configuration(\&check_network_card_setup_tabs);
+        record_info "check_network_card_setup_tabs";
+        verify_network_configuration(\&check_network_card_setup_tabs, "reload");
```

Comment:
I can only remember one reload when changing the device name, what is this reload for?

Reply:
Extending the filters to catch the reload, it showed that this method is also triggers a reload. As it was before this was not checked. it was only restart or not_restart